### PR TITLE
Adding changes for csv output

### PR DESCRIPTION
**Pull request by markcurtis1970**
This adds some changes to output all the lines to a csv file since the original `read_pandas` output only gave a truncated result set.

### CODE REVIEW

File: python/example.py [hunk 1]
```diff
@@ -81,11 +85,12 @@ def parse_arguments():
                         required=False, default=False)
     parser.add_argument('-certs', '--trustedCertificates', type=str,
       help='Path to trusted certificates for encrypted connection', required=False)
+    parser.add_argument('-output', '--output-file', type=str, help='output file', required=False, default='output.csv')
     return parser.parse_args()
 
 
 def connect_to_dremio_flight_server_endpoint(hostname, flightport, username, password, sqlquery,
-  tls, certs, disableServerVerification):
+  tls, certs, disableServerVerificationi, output_file):
```
Comment (jduo) [inline python/example.py:93]: There's an extra "i" after disableServerVerification now.

File: python/example.py [hunk 2]
```diff
@@ -156,8 +161,13 @@ def connect_to_dremio_flight_server_endpoint(hostname, flightport, username, pas
 
             # Retrieve the result set as a stream of Arrow record batches.
             reader = client.do_get(flight_info.endpoints[0].ticket, options)
+            print(type(reader))
             print('[INFO] Reading query results from Dremio')
-            print(reader.read_pandas())
+            res_table=(reader.read_all())
```
Comment (jduo) [inline python/example.py:166]: Let's only do this if output_file is not None.

File: python/example.py (hunk 1 above)
Comment (markcurtis1970) [reply python/example.py]: Missed that! thanks, corrected

File: python/example.py [hunk 3]
```diff
@@ -81,11 +85,12 @@ def parse_arguments():
                         required=False, default=False)
     parser.add_argument('-certs', '--trustedCertificates', type=str,
       help='Path to trusted certificates for encrypted connection', required=False)
+    parser.add_argument('-output', '--output-file', type=str, help='output file', required=False, default='output.csv')
```
[the remark this answers is not in the document]
Comment (markcurtis1970) [reply python/example.py]: Ok changes made, thanks

**Review by jduo — APPROVED** [review]

**Pull request merged**